### PR TITLE
Preconfirmations - new article

### PR DESCRIPTION
**Pull request by thogiti**
## Wiki PR Checklist

Thank you for contributing to the Protocol Wiki! Before you open a PR, make sure to read [information for contributors](https://epf.wiki/#/contributing) and take a look at following checklist:

- [x] Explain Ethereum Based Preconfirmations.

### CODE REVIEW

**Comment by thogiti** [comment]
@taxmeifyoucan This PR is ready for a review and your blessing.

**Comment by thogiti** [comment]
FYI...I will write a separate article on Ethereum Based Sequencing with Preconfs. 

Preconfs is a general framework can be used for other types of transaction processing. Based sequencing is one real cool use case to make Ethereum and Rollups behave like United Chains of Ethereum.

File: docs/wiki/research/Preconfirmations/Preconfirmations.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+
+**Additional Considerations:**
+
+- **Fallback Mechanisms:** In case of unexpected issues or if the first preconfer fails to include the transaction, users may have fallback options, such as requesting promises from multiple preconfers in parallel.
+
+- **Dispute Resolution:** The system may include mechanisms for resolving disputes in cases where there's disagreement about whether the promise was adequately fulfilled.
+
+
+## References
```
Comment (taxmeifyoucan) [inline docs/wiki/research/Preconfirmations/Preconfirmations.md:151]: Some recent resources
https://notes.ethereum.org/@JustinDrake/rJ2eXRcKa
https://www.youtube.com/watch?v=2IK136vz-PM

**Review by taxmeifyoucan — COMMENTED** [review]
Thanks a lot @thogiti, another great article :) Preconfs are an interesting and pretty recent topic, it's great to have it properly documented

File: docs/wiki/research/Preconfirmations/Preconfirmations.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+## [Overview](#overview)
+
+Based preconfirmations (preconfs) represent a significant advancement in Ethereum transaction processing, offering users swift and reliable execution. Through a combination of on-chain infrastructure, proposer accountability mechanisms, and flexible promise acquisition processes, preconfs stand to significantly enhance the user experience in Ethereum interactions. This technology not only reduces transaction latency but also introduces a layer of security and efficiency previously unseen in the ecosystem[^1].
+
+
+## [Construction of Preconf Promises](#construction-of-preconf-promises)
+
+Preconfirmation promises, or "preconfs," rely on two foundational on-chain infrastructure components:
+
+- **Proposer Slashing:** Proposers can opt into additional slashing conditions to ensure reliability and accountability. This approach draws inspiration from EigenLayer's model, which employs restaking as a means of enforcing these slashing mechanisms.
```
[the remark this answers is not in the document]
Comment (thogiti) [reply docs/wiki/research/Preconfirmations/Preconfirmations.md:13]: Yes, thx. Adding more relevant references in the next commit.

File: docs/wiki/research/Preconfirmations/Preconfirmations.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+- **Proposer Forced Inclusions:** To ensure the seamless execution of transactions, proposers have the authority to mandate the inclusion of specific transactions on-chain. This power is crucial in situations where the separation between proposers and builders (PBS) renders self-building uneconomical. The implementation of this mechanism typically involves the use of inclusion lists.
+
+
+When a L1 proposer decides to become a "preconfer," they are essentially agreeing to adhere to two distinct slashing conditions related to preconf promises. In return for their service, preconfers issue signed promises to users and are compensated with tips for successfully fulfilling these promises. The hierarchy among preconfers is determined based on their position in the slot order, with precedence given to those with earlier slot assignments.
```
[the remark this answers is not in the document]
Comment (thogiti) [reply docs/wiki/research/Preconfirmations/Preconfirmations.md:18]: Thx.

File: docs/wiki/research/Preconfirmations/Preconfirmations.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+- **Negative Tips:** Preconfers may accept negative tips for transactions that enhance their MEV opportunities, such as those affecting DEX prices and creating arbitrage prospects.
+
+
+Each of these elements plays a crucial role in the functionality and efficiency of based preconfirmations, ensuring transactions are not only processed swiftly but also securely and fairly within the Ethereum ecosystem.
+
+
+## [Preconfs Acquisition Process Flow](#preconfs-acquisition-process-flow)
+
+
+![Promise acquisition flow](https://ethresear.ch/uploads/default/original/2X/9/9667dc80c8d911fa3cd86108c3375d0de06e4252.png)
```
[the remark this answers is not in the document]
Comment (thogiti) [reply docs/wiki/research/Preconfirmations/Preconfirmations.md:67]: Ok, thx.

File: docs/wiki/research/Preconfirmations/Preconfirmations.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+- **Latency:** Utilizing direct communication channels, the process aims to achieve preconfirmation times as swift as 100 milliseconds, ensuring rapid transaction handling.
+
+- **Bootstrapping:** A substantial participation rate among L1 validators as preconfers is crucial. This ensures that within the proposer lookahead window, there is always a significant chance of encountering a preconfer ready to issue promises.
+
+- **Liveness Fallback:** Users can enhance their transaction's reliability by securing promises from several preconfers, thus safeguarding against the possibility of any single preconfer failing to fulfill their promise due to missed slots.
+
+- **Parallelization:** The system accommodates various promise types, ranging from strict commitments on post-execution state to more flexible, intent-based promises.
+
+- **Replay Protection:** Ensures transactions are protected from replay attacks, vital for maintaining the integrity and security of preconf transactions.
```
[the remark this answers is not in the document]
Comment (thogiti) [reply docs/wiki/research/Preconfirmations/Preconfirmations.md:48]: In the one of the document Justin explained a bit more on this. 

Replay protection in Ethereum Preconfs involves a combination of transaction uniqueness, cryptographic binding of preconf conditions to transactions, verification of preconf conditions before tx is included in the block, and rigorous enforcement mechanisms.

The original article [based preconfirmations](https://ethresear.ch/t/based-preconfirmations/17353) explains this a bit.

**Pull request merged**